docs/using-raiden-on-testnet/use-custom-token/README.rst

```diff
@@ -0,0 +1,12 @@
+Use Custom Token
```

Comment:
This whole tutorial does not seem to be reachable from anywhere?

Reply:
Yes, I didn't do the whole "use-custom-token" part yet and only included it by accident. I'll remove it from this PR and move it to a followup PR.